Documentation/release-notes/aapt.md

```diff
@@ -0,0 +1,19 @@
+#### Deprecation of aapt in favor of aapt2
+
+* warning XA1026: Using 'aapt' is deprecated in favor of 'aapt2'.
+  Please enable 'Use incremental Android packaging system (aapt2)' in
+  the Visual Studio project property pages or edit the project file in
+  a text editor and set the 'AndroidUseAapt2' MSBuild property to
+  'true'.
```

Comment:
As a tiny formatting idea to match some similar previous release notes, this could use a code block for the example warning:

````markdown
#### Deprecation of AAPT in favor of AAPT2

```
warning XA1026: Using AAPT is deprecated in favor of AAPT2. Please enable 'Use incremental Android packaging system (aapt2)' in the Visual Studio project property pages or edit the project file in a text editor and set the 'AndroidUseAapt2' MSBuild property to 'true'.
```
````

(Tentatively incorporating the other naming changes from other comments in this PR review.)